integ-test/src/test/java/org/opensearch/sql/legacy/CursorIT.java

```diff
@@ -465,6 +483,13 @@ public String executeFetchAsStringQuery(String query, String fetchSize, String r
     return responseString;
   }
 
+  private void verifyIsV1Cursor(String cursor) {
+    if (cursor.isEmpty()) {
+      return;
+    }
+    assertTrue("The cursor '" + cursor + "' is not from v1 engine.", cursor.startsWith("d:"));
```

Comment:
In legacy cursor can starts with `d:`, `j:` and `a:` https://github.com/Bit-Quill/opensearch-project-sql/blob/114e02a20f66fcab38b5df0c087ba1792da2f2c5/legacy/src/main/java/org/opensearch/sql/legacy/cursor/CursorType.java#L19-L21
```suggestion
    assertTrue("The cursor '" + cursor + "' is not from v1 engine.", !cursor.startsWith("n:"));
```

Reply:
Legacy only ever produces default cursors. Aggregation and Join enum values are not used.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/245/commits/cb97721147571ebad13c0017bb56851ef35b946d